irohad/consensus/yac/impl/cluster_order.cpp

```diff
@@ -34,8 +34,8 @@ namespace iroha {
           std::vector<std::shared_ptr<shared_model::interface::Peer>> order)
           : order_(std::move(order)) {}
 
-    // TODO :  24/03/2018 x3medima17: make it const, IR-1164
-    const shared_model::interface::Peer& ClusterOrdering::currentLeader() {
+      // TODO :  24/03/2018 x3medima17: make it const, IR-1164
```

Comment:
maybe fix that here as well? it take ~3loc, i guess

Reply:
No, because there is changes the state inside. We can fix the issue in the separate PR.